contracts/governance/TimelockController.sol

```diff
@@ -199,6 +177,22 @@ contract TimelockController is AccessControl, ERC721Holder, ERC1155Holder {
         return _timestamps[id];
     }
 
+    /**
+     * @dev Returns operation state.
+     */
+    function getOperationState(bytes32 id) public view virtual returns (OperationState) {
+        uint timestamp = getTimestamp(id);
+        if (timestamp == 0) {
+            return OperationState.Unset;
+        } else if (getTimestamp(id) > _DONE_TIMESTAMP && timestamp > block.timestamp) {
+            return OperationState.Pending;
+        } else if (timestamp > _DONE_TIMESTAMP) {
+            return OperationState.Ready;
+        } else {
+            return OperationState.Done;
+        }
+    }
```

Comment:
We can reduce the number of checks, an avoid a duplicated `getTimestamp` call
```suggestion
        if (timestamp == 0) {
            return OperationState.Unset;
        } else if (timestamp == DONE_TIMESTAMP) {
            return OperationState.Done;
        } else if (timestamp > block.timestamp) {
            return OperationState.Pending;
        } else {
            return OperationState.Ready;
        }
    }
```

Reply:
Note that this slightly changes the semantics:

Before:
- Pending: is going to be executed at some point, may or may not be ready
- Ready: can be executed now

After:
- Pending: scheduled but now yet ready for execution
- Ready: can be executed now


So before Ready implied Pending, and after Ready implies not Pending.
If we keep the old getters, we will need to either change their semantics, or include an inconsistency between what they return and what is returned by state().

Reply:
I suggest the name 'pending' be replaced by 'blockded' or something similar